assets/feature-installer-utils.sh

```diff
@@ -1,4 +1,31 @@
-# Execute a command and if it fails, return its output and exit
+# Set some informative variables
+
+# The flavour we're running on
+FLAVOUR="$(cat /home/cloudcontrol/flavour)"
+export FLAVOUR
+
+# The path to install software binaries to
+BINPATH="/home/cloudcontrol/bin"
+export BINPATH
```

Comment:
Why split the export and the define of the variable in two lines?

Reply:
Because something advised me to do it. I thought it was Shellcheck, but it doesn't complain anymore. I'll correct it.

Reply:
Ahhh. It was this one: https://github.com/koalaman/shellcheck/wiki/SC2155 But it's only relevant to line 4.

Reply:
Right, because exit values from the command might be suppressed.